src/Lykke.AlgoStore.CSharp.AlgoTemplate.Models/Models/StatisticsSummary.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Lykke.AlgoStore.CSharp.AlgoTemplate.Models.Enumerators;
+
+namespace Lykke.AlgoStore.CSharp.AlgoTemplate.Models.Models
+{
+    public class StatisticsSummary
+    {
+        [Required]
+        public string InstanceId { get; set; }
+
+        public int TotalNumberOfTrades { get; set; }
+
+        public int TotalNumberOfStarts { get; set; }
+
+        public double InitialWalletBalance { get; set; }
+
+        public double LastWalletBalance { get; set; }
+
+        public double AssetOneBalance { get; set; }
+
+        public double AssetTwoBalance { get; set; }
+
+        public double NetProfit => System.Math.Abs(InitialWalletBalance) > 0 ? (InitialWalletBalance - LastWalletBalance) / InitialWalletBalance : 0;
```

Comment:
I think NetProfit will be calculated every time. When we have the "Live" type algo instance, of the wallet will be changed depending on the price - which can be each second, minute. So let's stay this way for the "back-test" but for "Live" I think we should calculate it every time.